core/test/unit/migration_spec.js

```diff
@@ -13,6 +13,11 @@ var should = require('should'), // jshint ignore:line
 
     sandbox = sinon.sandbox.create();
 
+// Clone the fixtures as they get changed during the migration.
+// The initial blog posts will be added a `published_at` property, which
+// would change the fixturesHash.
+fixtures = _.cloneDeep(fixtures);
```

Comment:
Hmm can we clone the fixtures.json inside `core/server/data/schema/fixtures/utils.js`?
This utility is actively modifying the fixtures. And imagine your write different test file testing something with fixtures, you will run into the same problem again,

Reply:
@AileenCGN I would like to merge this soon. Could you please reply the next days? Thanks 🙏🏽

Reply:
@kirrg001 I am sorry, that I didn't reply to this earlier. I will try this again asap. I do remember that your recommended way didn't work for me, but as you're not able to reproduce it, I'm probably doing something wrong.

Reply:
@kirrg001 Removing the cloning of the fixtures in `migration_spec` and moving it to the fixtures utils, results in the fixtures hash getting recalculated which every test. Keeping both, works fine (manual and coded tests).

I pushed the changes for you to check and maybe reproduce what I am saying.